### PR TITLE
1stand2ndweekUpdates

### PR DESCRIPTION
**Pull request by Prakhar-commits**
Quiz Creator

This PR consists of the updates for the first two weeks of the C4GT program

It consists of the weekly updates

### CODE REVIEW

File: docs/2023/Quiz Creator/Quiz Creator/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/Quiz Creator/Quiz Creator
+---
+
+
+# Author
+Rishabh Jain
```
Comment (Shruti3004) [inline docs/2023/Quiz Creator/Quiz Creator/index.md:9]: Hey @Prakhar-commits please add your name here

File: docs/2023/Quiz Creator/Quiz Creator/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Contact Info
+<!-- - [Email](mailto:) -->
+<!-- - [Linked In]() -->
+<!-- - [GitHub]() -->
```
Comment (Shruti3004) [inline docs/2023/Quiz Creator/Quiz Creator/index.md:14]: Please add these links

**Review by Shruti3004 — CHANGES_REQUESTED** [review]

File: docs/2023/Quiz Creator/Quiz Creator/updates/2023-07-14.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- [X] Initial Frotend consisting of multiple pages
+
+
+
+## Contributions
+[link][https://github.com/avantifellows/quiz-creator/pull/8]
+[link][https://github.com/avantifellows/quiz-creator/pull/9]
+
+## Learnings
+* Understanding of the proper ways to Raise a PR when working within a organisation
```
Comment (Apoorvgarg-creator) [inline docs/2023/Quiz Creator/Quiz Creator/updates/2023-07-14.md:17]: Learnings should be descriptive

**Review by Apoorvgarg-creator — CHANGES_REQUESTED** [review]

File: docs/2023/Quiz Creator/Quiz Creator/index.md (hunk 2 above)
Comment (Prakhar-commits) [reply docs/2023/Quiz Creator/Quiz Creator/index.md]: Done

File: docs/2023/Quiz Creator/Quiz Creator/index.md (hunk 1 above)
Comment (Prakhar-commits) [reply docs/2023/Quiz Creator/Quiz Creator/index.md]: Done

File: docs/2023/Quiz Creator/Quiz Creator/updates/2023-08-04.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: Week 5
+author: Rishabh Jain
```
Comment (wasup-yash) [inline docs/2023/Quiz Creator/Quiz Creator/updates/2023-08-04.md:3]: you need to update you name here too

Comment (Prakhar-commits) [reply docs/2023/Quiz Creator/Quiz Creator/updates/2023-08-04.md]: okay

File: docs/2023/Quiz Creator/Quiz Creator/updates/2023-07-14.md (hunk 3 above)
Comment (Prakhar-commits) [reply docs/2023/Quiz Creator/Quiz Creator/updates/2023-07-14.md]: okay

**Comment by GeekGawd** [comment]
@Prakhar-commits you still have conflicts please resolve them

**Comment by Prakhar-commits** [comment]
> @Prakhar-commits you still have conflicts please resolve them

These are vercel related conflicts cannot be resolved unless you have write access

**Review by wasup-yash — APPROVED** [review]

**Comment by GeekGawd** [comment]
Merged with #286

**Pull request merged**